client/blocks/app-banner/index.jsx

```diff
@@ -274,4 +347,4 @@ const mapDispatchToProps = {
 	dismissAppBanner,
 };
 
-export default connect( mapStateToProps, mapDispatchToProps )( localize( AppBanner ) );
+export default connect( mapStateToProps, mapDispatchToProps )( withRtl( localize( AppBanner ) ) );
```

Comment:
Not a blocker by any means, but, if desired, we could leverage `connect` to avoid the rightward drift of the nested function transformations. The approach is found in other modules. 

https://github.com/Automattic/wp-calypso/blob/e385f512e946afcf17c16034ebdcdfb4e690f0ac/client/my-sites/activity/filterbar/date-range-selector.jsx#L206-L210

Reply:
That looks so much cleaner! I've gone ahead to implement it in https://github.com/Automattic/wp-calypso/pull/67013/commits/4b4f68a8c39949db3c97196b96837e49eaf9bac3.